app/src/main/res/menu/card_view_menu.xml

```diff
@@ -2,6 +2,11 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:app="http://schemas.android.com/apk/res-auto">
 
+    <item
+        android:id="@+id/action_duplicate"
+        android:icon="@drawable/ic_duplicate"
+        android:title="@string/duplicateCard"
+        app:showAsAction="always" />
```

Comment:
I'd much prefer duplication to never be an action, as it's not a common action at all.

It may be worth moving sharing to the overflow menu too then so the overflow menu doesn't contain only a single entry, but I will probably do that myself separately from this PR so we can focus on a single change.

Reply:
Instead of using duplication as an action you mean putting a button or something like that would be better? Also i don't really get what do you mean when you say overflow menu, could you please explain that to me.

Reply:
Just use `app:showAsAction="never"` instead, Android will create an "overflow menu" (aka: kebab menu or "3 dots menu")

Reply:
Ok, i added this, sorry again for the mess with the branches